app/utils.py

```diff
@@ -29,3 +33,61 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    viewport_default = '39.861204,-75.310357|40.138932,-74.928582'
```

Comment:
Why do we give it two sets of coordinates?

Can we add some attribute in config that contains the viewport default? That way if we redeploy the app for other cities, we can change that config and all will still work.